src/common/WorkQueue.cc

```diff
@@ -122,6 +122,8 @@ void ThreadPool::worker(WorkThread *wt)
 	  processing++;
 	  ldout(cct,12) << "worker wq " << wq->name << " start processing " << item
 			<< " (" << processing << " active)" << dendl;
+          assert(strlen(wq->thread_name) < 16);
+          pthread_setname_np(wt->get_thread_id(), wq->thread_name);
```

Comment:
Oh.. I think it's better to name the thread after the thread pool (and not have a thread name property for work queues)

Also, we might consider having the thread pool name have a 13 char limit and appending a "/%d" to the end... not sure if that's useful or not?